helpers/helpers.v2.1.d/backup

```diff
@@ -88,7 +95,12 @@ ynh_backup() {
     fi
 
     # Add the relative current working directory to the destination path
-    local rel_dir="${YNH_CWD#"$YNH_BACKUP_DIR"}"
+    local rel_dir
+    if [ "${YNH_CORE_SCRIPT:-}" != 1 ]; then
+        rel_dir="${YNH_CWD#"$YNH_BACKUP_DIR"}"
+    else
+        rel_dir="${YNH_CORE_BACKUP_DIR#"$YNH_BACKUP_DIR"}"
```

Comment:
Use YNH_CWD instead of YNH_CORE_BACKUP_DIR like in https://doc.yunohost.org/admin/backups/include_exclude_files